pkg/controller/injector/k8s_binding_injector_test.go

```diff
@@ -17,11 +17,213 @@ limitations under the License.
 package injector
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
```

Comment:
I think the typical style is to keep standard go imports at the top and then "third party" separate below. e.g.,

import (
  "errors"
  "fmt"

  "github.com/..."
)